script.js

```diff
@@ -1,4 +1,186 @@
+var deck = shuffleCards(createDeck());
+var dealerCards = [];
+var playerCards = [];
+
+//check if Blackjack
+var isBlackjack = function(hand){
+  if(hand.length == 2 && cardsSum(hand) == 21){
+    return true;
+  }else {
+    return false;
+  }
+}
+
+//Deal card
+var dealCard = function (hand) {
+  hand.push(deck.pop());
+};
+
+//Get sum of cards
+var cardsSum = function (hand) {
+  var sum = 0;
+  var aceCount = 0;
+  for (card in hand){
+    if(card.name == 'ace'){
+      sum += 11;
+      aceCount += 1;
+    }else {
+      sum += card.number;
+    }
+  }
+
+  //if sum is more than 21
+  if (sum > 21 && aceCount > 0){
+    for(let x = 0; x < aceCount; x++){
+      sum -= 10;
+      if (sum <= 21){
+        break;
+      }
+    }
+    
+  }
+  return sum;
+}
+
+
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var myOutputValue = ``;
+
+  //Player deal cards when click submit button
+  if(input == ''){
+
+    if(playerCards.length === 0){
+      //First card
+      dealCard(playerCards);
+      dealCard(dealerCards);
+      //Second card
+      dealCard(playerCards);
+      dealCard(dealerCards);
+
+      myOutputValue = `Your cards are ${playerCards}. Please type 'hit' to draw more cards, or 'stand' to end your turn`;
+
+      //if Blackjack
+      if (isBlackjack(playerCards)){
+        myOutputValue = `Blackjack! Your cards are ${playerCards}.You've won! Click submit to play again.`;
+        resetGame();
+      }
+    }
+  }
+
+  //if player wants to draw more cards
+  if(input == `hit`){
+    dealCard(playerCards);
+    if(cardsSum(playerCards) > 21){
+      myOutputValue = `You lose! Your cards are ${playerCards}. Click submit to play again.`;
+      resetGame();
+    }else{
+      myOutputValue = `You drew a new card. Your cards are ${playerCards}. Please type 'hit' to draw more cards, or 'stand' to end your turn.`;
+    }
+  }
+  //if player does not want to draw more cards
+  if(input == `stand`){
+    if(cardsSum(dealerCards) <= 16){
```

Comment:
You can use a while loop here instead in case the dealer still has less than 16 points after drawing 1 card :)